packages/react-components/review/beta/react-components.api.md

```diff
@@ -1251,6 +1220,55 @@ export const _LocalVideoTile: React_2.MemoExoticComponent<(props: {
     personaMinSize?: number | undefined;
 }) => JSX.Element>;
 
+// @beta
+export interface Mention {
+    displayText: string;
+    icon?: JSX.Element;
+    id: string;
+}
+
+// @beta
+export interface MentionDisplayOptions {
```

Comment:
Are we not giving developers the ability to change the highlighted name's colour or font styling at all?

Reply:
We are; through CSS styling and a custom renderer.